Stylesheets/utility.css

```diff
@@ -0,0 +1,38 @@
+:root {
+    --flex-gap : 1rem;
+
+    /* border radius */
+    --container-border: 0.3rem;
+
+
+    /* transitions */
+    --hover : 0.5s
+}
+
+body {
+    height:100vh;
+    margin: 0 auto;
+}
```

Comment:
here, height: 100vh, wouldn't it overflow when the content increases?

Reply:
oh understood, I'll rectify it👍